src/ObjectUtils/CoffFile.cpp

```diff
@@ -548,6 +550,24 @@ ErrorMessageOr<std::vector<CoffFileImpl::UnwindRange>> CoffFileImpl::GetUnwindRa
   return unwind_ranges;
 }
 
+ErrorMessageOr<orbit_grpc_protos::ModuleSymbols> CoffFileImpl::LoadRuntimeFunctionsAsSymbols() {
+  ErrorMessageOr<std::vector<UnwindRange>> unwind_ranges_or_error = GetUnwindRanges();
+  if (!unwind_ranges_or_error.has_value()) {
+    return unwind_ranges_or_error.error();
+  }
+
+  ModuleSymbols module_symbols;
+  for (const UnwindRange& unwind_range : unwind_ranges_or_error.value()) {
+    SymbolInfo* symbol_info = module_symbols.add_symbol_infos();
+    // Let's assign an arbitrary function name right away, as we want a non-empty and unique name in
+    // many places.
+    symbol_info->set_demangled_name(absl::StrFormat("[function@%#x]", unwind_range.start));
```

Comment:
Its a range, shall we do `function@start-end`?

Reply:
I don't know, I wouldn't really like something like `[function@0x180086400-0x180086400]`. I think just the start is fine for a meaningful and unique name.

Reply:
I think both would be fine. I was wandering about the [unknonw@0x123] case, where this is a single address. So is it clear enough that this is a range here?

Reply:
>I'm wondering if it's more about [unknown@0x180086400] not being clear enough.

I think this is actually the case. Let's keep `function@0x123412`. We can adjust the `unknown` case, later.

Reply:
The ideal situation would be that this fallback works so well that the unknown case becomes very rare. Just wishful thinking of course.

Reply:
Valid point.